CONTRIBUTING.md

```diff
@@ -87,9 +87,18 @@ Your contribution needs to meet the following standards:
   Co-authored-by: <B full name> <B email>
   ```
 
-- Usage of `unsafe` should be documented with a justification (over its safe
-  alternative where one exists) and an explanation on why it cannot produce
-  undefined behavior. E.g.
+- **Usage of `unsafe` is heavily discouraged**. If `unsafe` is required,
+  it should be accompanied by a comment detailing its...
+  - Justification, potentially including quantifiable reasons why safe
+    alternatives were not used (e.g. via a benchmark showing a valuable
+    performance improvements), and
```

Comment:
I don't think it reads well to include `, and` in the bullet point. I would tend to prefer treating the bullets points as alternate endings to the beginning sentence, rather than one continuous sentence.

Reply:
done